Standard.AI.Data.EntityIntelligence/Models/Datas/TableInformation.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Standard.AI.Data.EntityIntelligence.Models.Datas
+{
+    internal class TableInformation
+    {
+        public string Name { get; set; }
+        public string Schema { get; set; }
+        public IEnumerable<TableColumn> Columns { get; set; }
```

Comment:
Seems inconsistent naming. 
`TableInformation` and `TableColumn`

Either we go with the information suffix or not? 
Since column is already in the context of the table object we don't need prefix Table.
Also should Schema be called `SchemaName`

So why not going with:
``` csharp
 internal class TableInformation
    {
        public string Name { get; set; }
        public string Schema { get; set; }
        public IEnumerable<ColumnInformation> Columns { get; set; }
  }
```

We can also go with Metadata instead of Information since metadata is the actual thing we're dealing with